problem10_findPeekElement.java

```diff
@@ -0,0 +1,38 @@
+package BinarySearch2;
+
+public class problem10_findPeekElement {
+    /*
+    Brute Force ---->
+        Time Complexity : O(n)
+        Space Complexity: O(1)
+     */
+    public int findPeakElement(int[] nums) {
+        if(nums == null || nums.length == 0) return -1;
+        int low = 0;
+        int high = nums.length - 1;
+
+        while(low <= high) {

```

Comment:
Make sure to include the time and space complexity for the binary search approach as well